src/client/eppo-client.ts

```diff
@@ -756,14 +756,14 @@ export default class EppoClient {
 
         try {
           this.logBanditAction(banditEvent);
-        } catch (err: any) {
-          logger.error('Error logging bandit event', err);
+        } catch (err) {
+          logger.error({ err }, 'Error logging bandit event');
```

Comment:
Pino's API ([link](https://getpino.io/#/docs/api?id=loggererrormergingobject-message-interpolationvalues)) is to have context _first_, then the log message.